v3.0/how-to/deploy/from-tarball/production-environment.md

```diff
@@ -107,15 +107,14 @@ See the following table for the default ports for the optional TiDB components:
 
 ```
 # Download the package.
-$ wget http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz
-$ wget http://download.pingcap.org/tidb-latest-linux-amd64.sha256
+$ wget http://download.pingcap.org/tidb-v3.0-linux-amd64.tar.gz http://download.pingcap.org/tidb-v3.0-linux-amd64.sha256
```

Comment:
The default link should be `http://download.pingcap.org/tidb-v3.0.0-linux-amd64.tar.gz` and `http://download.pingcap.org/tidb-v3.0.0-linux- Amd64.sha256`

Reply:
@liubo0127 I disagree with that. I don't think we should refer to specific point versions in the documentation, because it means we have to update the documentation each time there's a new point release. The v3.0 package always contains the most recent stable point release of TiDB 3.0, so we can safely use that version in the documentation.

Reply:
I had some discussion elsewhere on this point with @lilin90 and @queenypingcap, maybe they can help us resolve this.

Reply:
If just let the user use the latest version, there is no need to modify.